### PR TITLE
Adds endpoint documentation to readme

### PR DESCRIPTION
**Pull request by rolandpoulter**
#35

### CODE REVIEW

**Comment by rolandpoulter** [comment]
@king-jam would you mind reading over this and letting me know if it makes sense?

File: README.md [hunk 1]
```diff
@@ -22,6 +22,14 @@ By default there will be two servers running. One on port `5000` which will be p
 
  * http://localhost:3000 is the MonoRail Web UI.
 
+### How to configure RackHD API endpoint.
+
+You can change the RackHD API endpoint dynamically from the UI. There is a gear icon located at the top right corner of the UI. This will open the settings panel where you can type a custom Rack API endpoint URL.
+
+The default API endpoint can be configured by renaming `apps/monorail/custom.json.example` and editing the `MONORAIL_API` property.
```
Comment (heckj) [inline README.md]: renaming to `apps/monorail/custom.json`?

Comment (rolandpoulter) [inline README.md]: I was thinking it would be obvious, but I'll make it more explicit.

**Comment by king-jam** [comment]
:+1:

**Pull request merged**